core.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+var time_interval = 500;
+var counter = 0, ok_value = 0, ng_value = 0;
+
+function changeFileName() {
+    var image1 = new Image();
+    var image2 = new Image();
+    var imageName1 = counter + "_OK.jpg";
+    var imageName2 = counter + "_NG.jpg";
+    image1.src = imageName1;
+    if (image1.width === 0) { //there is no ok image
+        image2.src = imageName2; 
+        if (image2.width !== 0) { //there is ng image
+            document.getElementById("Img_in").src = imageName2;
+            document.getElementById("Img_in").style.border = "thick solid #FF0000";
+            document.getElementById("Img_in").style.height = "200px";
+            document.getElementById("Img_in").style.width = "300px";
+            document.getElementById("Result_text").innerHTML = "NOTGOOD";

```

Comment:
To follow strictly the problem statement, can we edit this into `"NG (Not good)"` anh?

Reply:
Sure. I also forgot that in case you want picture to be full size, you can actually delete those two lines
document.getElementById("Img_in").style.height = "200px";
document.getElementById("Img_in").style.width = "300px";